### PR TITLE
Avoid TEC dependency in orders page

### PR DESCRIPTION
**Pull request by dpanta94**
### 🎫 Ticket

[ET-2117]

### 🗒️ Description

Avoid Dependency on TEC. The date range dropdowns were only being initiated while TEC was active. Now ET can initiate those on its own.

### ✔️ Checklist
- [ ] Changelog entry in the `readme.txt` file.
- [x] Code is covered by **NEW** `wpunit` or `integration` tests.
- [x] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [x] Are all the **required** tests passing?
- [x] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

[ET-2117]: https://stellarwp.atlassian.net/browse/ET-2117?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tickets/Commerce/Admin/Orders_Page.php [hunk 1]
```diff
@@ -164,4 +180,27 @@ public function is_admin_orders_page() {
...
 	}
+
+	/**
+	 * Checks if the current screen is the orders page and the Tickets Events Calendar is not active.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_admin_orders_page_and_no_TEC() { // phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
```
Comment (stratease) [inline src/Tickets/Commerce/Admin/Orders_Page.php:191]:
```suggestion
	public function is_admin_orders_page_and_no_TEC(): bool { // phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
```

File: src/Tickets/Commerce/Admin/Orders_Page.php [hunk 2]
```diff
@@ -164,4 +180,27 @@ public function is_admin_orders_page() {
...
+		if ( ! $this->is_admin_orders_page() ) {
+			return false;
+		}
+
+		try {
+			tribe( 'tec.main' );
```
Comment (stratease) [inline src/Tickets/Commerce/Admin/Orders_Page.php:197]:
```suggestion
			tribe( 'tec.main' );
```
Not a huge fan of using a try catch to validate TEC is installed. I'd prefer looking for a TEC supported constant. And not touching class_exists() checks either, that can be troublesome.

Thoughts?

**Review by stratease — CHANGES_REQUESTED** [review]
Minor things.

File: src/Tickets/Commerce/Admin/Orders_Page.php (hunk 2 above)
Comment (dpanta94) [reply src/Tickets/Commerce/Admin/Orders_Page.php:197]: Yea i think constant would definitely be faster and cleaner. Bad choice evaluation through try-catch for sure looking at it now.

Thanks @stratease !

**Review by redscar — APPROVED** [review]

**Pull request merged**